src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/ClusterComponent.scala

```diff
@@ -111,6 +111,8 @@ trait ClusterComponent extends LeoComponent {
 
   object clusterQuery extends TableQuery(new ClusterTable(_)) {
 
+    def save(cluster: Cluster): DBIO[Cluster] = save(cluster, initBucket = None, serviceAccountKeyId = None)
```

Comment:
Any downside to just adding the default args to the existing `save` method? E.g.
```
def save(cluster: Cluster, initBucket: Option[GcsPath] = None, serviceAccountKeyId: Option[ServiceAccountKeyId] = None): DBIO[Cluster]
```